about/pricing.html.markerb

```diff
@@ -67,12 +67,85 @@ We charge for started and stopped Machines differently. Attached GPUs are charge
 
 ### Started Fly Machines
 
-The price of a running [Fly Machine](/docs/machines/) VM is the price of a named CPU/RAM preset, plus about $5 per 30 days per GB of additional RAM.
-
-Here's the pricing for named presets and the allowed additional RAM configurations:
+<% if Time.now.utc < '2024-11-01T00:00:00Z' %>
+<section class="callout">
+  **Machines pricing is changing!**
+
+  <p class="mt-2">
+    On July 1st, we're beginning a phased roll-out of per-region pricing. For the first month, prices will remain the same, but the lines on your invoice will be broken down by region. Each subsequent month, prices will be increase by 25% of the difference between the price in `iad` and the price listed for the region, until the listed price is reached in November.
+  </p>
+
+  <p class="my-4 w-full text-right"><button id="phased-roll-out-example-toggle-btn" toggle>See how the phased roll-out works</button></p>
```

Comment:
Since the example is static, I would clarify that it's an example, something like
```suggestion
  <p class="my-4 w-full text-right"><button id="phased-roll-out-example-toggle-btn" toggle>See an example of how the phased roll-out works</button></p>
```